opentelemetry-sdk/src/metrics/manual_reader.rs

```diff
@@ -84,9 +84,7 @@ impl MetricReader for ManualReader {
             if inner.sdk_producer.is_none() {
                 inner.sdk_producer = Some(pipeline);
             } else {
-                global::handle_error(MetricsError::Config(
-                    "duplicate reader registration, did not register manual reader".into(),
-                ))
+                otel_error!(name: "ManualReader.RegisterPipeline.DuplicateRegistration");
```

Comment:
info/debug only. Even if a user gets this message, they won't know what to do. Its helpful to us only.